### PR TITLE
dropping references to vm lifecycle events

### PR DESCRIPTION
**Pull request by kbrock**
part of https://github.com/ManageIQ/manageiq/issues/21204

I did a quick search.
Just wanted to see if we could move this conversation forward

### CODE REVIEW

**Review by jrafanie — APPROVED** [review]
LGTM, should be merged with https://github.com/ManageIQ/manageiq-api/pull/1232

File: policies_and_profiles_guide/_topics/chap_events.md [hunk 1]
```diff
@@ -11,7 +11,6 @@ Events are triggers that cause a condition to be tested. Control provides severa
 | Orchestration Lifecycle   | Events related to orchestration lifecycle, such as retirement.                                                                                                                                       |
 | Physical Server Operation | Events related to the connection state of a physical server.                                                                                                                                         |
 | VM Configuration          | Events associated with a change in configuration of a virtual machine. These include, but are not limited to, clone, create, template create, and settings change.                                   |
-| VM Lifecycle              | Events such as virtual machine discovery, provisioning, and virtual machine retirement.                                                                                                              |
```
Comment (Fryguy) [inline policies_and_profiles_guide/_topics/chap_events.md:14]: This should not be removed.  We do still have the lifecycle events, we just don't have it as a dedicated model.

File: provisioning_virtual_machines_and_hosts/_topics/cfme_lifecycle.md
Comment (Fryguy) [inline provisioning_virtual_machines_and_hosts/_topics/cfme_lifecycle.md:1]: Same here, this should not be removed. (Though it being named cfme_lifecycle is wrong :) )

**Comment by kbrock** [comment]
update:
- added cfme_lifecycle.md and TOC link back
- left note in chapter event about add lifecycle event back

I'm confused. Is any of this still good or should I just drop this PR?

**Review by Fryguy — APPROVED** [review]

**Pull request merged**